### PR TITLE
Use `idx` as primary key to ensure uniqueness.

### LINKED ISSUE #612: Report pagination error
You get half the reports you ask for 
example: /committee/C00048587/reports?per_page=4

### PR DESCRIPTION
**Pull request by jmcarp**
Based on discussion with @LindsayYoung and @arowla. Note: I changed `one` to `first` for committee lookups in the reports and totals views, since I was running into duplicate key errors while debugging this. We can think about a better solution later on, but I think this is fine for now.

[Resolves #612]

### CODE REVIEW

File: webservices/resources/reports.py [hunk 1]
```diff
@@ -531,5 +532,6 @@ def get_reports(self, committee_id, reports_class, args, page_num, per_page):
             reports = reports.filter(reports_class.cycle.in_(args['cycle'].split(',')))
 
         count = reports.count()
+        import ipdb; ipdb.set_trace()
```
Comment (arowla) [inline webservices/resources/reports.py]: stray debugger ^^

Comment (jmcarp) [inline webservices/resources/reports.py]: Fixed :confounded:

**Comment by arowla** [comment]
Looks good, just waiting on Travis.

**Comment by arowla** [comment]
Wait, Travis won't pass until #635 is merged. Nevermind.

**Comment by jmcarp** [comment]
Replaced by #653.

**Pull request closed without merging**